e2e_tests/src/e2e_conduit/subslurp.py

```diff
@@ -0,0 +1,60 @@
+from datetime import datetime, timedelta
+import gzip
+import io
+import logging
+import re
+
+logger = logging.getLogger(__name__)
+
+
+class subslurp:
+    # accumulate stdout or stderr from a subprocess and hold it for debugging if something goes wrong
```

Comment:
nit: or is this form only for functions?
```suggestion
    """ accumulate stdout or stderr from a subprocess and hold it for debugging if something goes wrong """
```